docs/xml/enumerations/vote_variation.rst

```diff
@@ -1,35 +1,43 @@
 VoteVariation
 =============
 
-.. todo::
-
-   Document VoteVariation
 
 +----------------------+----------------------------------------------------------------------------------+
 | Name                 | Description                                                                      |
 |                      |                                                                                  |
 +======================+==================================================================================+
-| 1-of-m               |                                                                                  |
-+----------------------+----------------------------------------------------------------------------------+
-| approval             |                                                                                  |
+| 1-of-m               | A method where each voter can select up to one candidate.                        |
 +----------------------+----------------------------------------------------------------------------------+
-| cumulative           |                                                                                  |
+| approval             | `Approval voting`_, where each voter can select as many candidates as desired.   |
 +----------------------+----------------------------------------------------------------------------------+
-| majority             |                                                                                  |
+| cumulative           | `Cumulative voting`_, where each voter can distribute their vote to up to        |
+|                      | *N* candidates.                                                                  |
 +----------------------+----------------------------------------------------------------------------------+
-| n-of-m               |                                                                                  |
+| majority             | A 1-of-m method where the winner needs more than 50% of the vote to be elected.  |
```

Comment:
Yes, there is single-winner RCV for example (in the final round of the tabulation).  But for that case, I would hope we want them to put "rcv" rather than "majority".

As for others, I don't know.  But the problem is that if there is, how would the jurisdiction know if they should label it "majority" or the other thing?  (When I asked, you said that feed creators were only allowed to choose one enum value per contest as opposed to multiple.)

Given that, I would rather us resolve the confusion for them (so it's obvious what enum value they need to choose) as opposed to leaving things ambiguous and so uncertain as to what what enum they're supposed to pick.